src/Modules/LearningTasks/Tutor.LearningTasks.Infrastructure/Tutor.LearningTasks.Infrastructure.csproj

```diff
@@ -6,6 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Dahomey.Json" Version="1.12.2" />
```

Comment:
If you manage to migrate most of the configuration to `BuildingBlocks.Infrastructure`, check if you can remove this dependency.